utils/src/test/java/com/cloud/utils/ScriptTest.java

```diff
@@ -111,6 +111,20 @@ public String interpret(BufferedReader reader) throws IOException {
         Assert.assertNotNull(value);
     }
 
+    @Test
+    public void executeWithOutputInterpreterAllLinesParserLargeOutput() {
+        Assume.assumeTrue(SystemUtils.IS_OS_LINUX);
+        OutputInterpreter.AllLinesParser parser = new OutputInterpreter.AllLinesParser();
+        Script script = new Script("seq");
+        script.add("-f");
+        script.add("my text to test cloudstack %g");
+        script.add("4096"); // AllLinesParser doesn't work with that amount of data
+        String value = script.execute(parser);
+        // it is a stack trace in this case as string
+        Assert.assertNull(value);
+        Assert.assertEquals(129965, parser.getLines().length());
```

Comment:
@rp- how can you be sure of this value?

Reply:
well, it will always create the same text? see:
```
seq -f "my text to test cloudstack %g" 4096 | wc -c
129965
```

Reply:
ah, missed the meaning of the command :+1: 
(must say beauty is not this test's purpose ;)